package.json

```diff
@@ -2,7 +2,12 @@
   "name": "",
   "version": "1.0.0",
   "scripts": {
-    "test": "elm make --docs=docs.json && elm-format src/ tests/ --validate && elm-test && elm-review && npm run package-tests -s",
+    "test": "npm-run-all -l -s test:*",
```

Comment:
Can you turn `-l` and `-s` into their more verbose counterparts? I believe this would be clearer to people who want to know what is happening?

I would also think that it would be clearer and to list the scripts to run rather than using `test:*`. Right now, the script looks very magical and it can be hard to deduce what it does without running it or having knowledge of `npm-run-all`.

Reply:
You mean, the long form of each option `--print-label` and `--sequential`? No PB.

The wildcard syntax is very common with `npm-run-all` but I agree this looks puzzling to newcomers that don't know that the order of the scripts matter (that's a feature actually, so that you can add new sub-scripts easily). If only `package.json` had comments... No PB here as well.

Reply:
Yes, I meant the long form of each option.

I agree that a comment could have been useful. But I think that a more explicit version will be nicer. Especially since I will have this be the default for packages created by `elm-review new-package`, and users who use that will then be the ones who need to maintain these scripts. I don't want them to be lost because we tried to make the tools too smart.